packages/assets/extract-svgs.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Extracts icons from VADS icon sprite as single svgs
+ * Adapted from https://gist.github.com/mhmdnsr/18343cceb12858c8b148ef31d1e5a32b
+ */
+
+import * as fs from 'fs'
+import { parse } from 'svg-parser'
+import { toHtml } from 'hast-util-to-html'
+
+fs.readFile(
+  '../../node_modules/@department-of-veterans-affairs/component-library/dist/img/sprite.svg',
```

Comment:
Is your local client out of sync or something here? It doesn't appear that the `@department-of-veterans-affairs/component-library` is a dependency of this or any of our packages so it seems like this file shouldn't exist.